dev/com.ibm.ws.jaxrs.2.0.common/src/com/ibm/ws/jaxrs20/endpoint/AbstractJaxRsWebEndpoint.java

```diff
@@ -132,6 +135,9 @@ public void invoke(HttpServletRequest request, HttpServletResponse response) thr
         try {
             updateDestination(request);
             destination.invoke(servletConfig, servletConfig.getServletContext(), request, response);
+        } catch (InvalidCharsetException e) {
+                response.setStatus(HttpServletResponse.SC_UNSUPPORTED_MEDIA_TYPE);
+                return;
```

Comment:
Since you did a separate exception catch (awesome), the return is redundant.

Reply:
fair point, the separate catch was for the FFDCIgnore to work. I'll remove the return.